pkg/compose/pull.go

```diff
@@ -130,6 +130,13 @@ func (s *composeService) pull(ctx context.Context, project *types.Project, opts
 					mustBuild = append(mustBuild, service.Name)
 				}
 				if !opts.IgnoreFailures && service.Build == nil {
+					if s.dryRun {
+						w.Event(progress.Event{
+							ID:     service.Name,
+							Status: progress.Error,
+							Text:   fmt.Sprintf(" - Pull error for image: %s", service.Image),
```

Comment:
Should we also show the contents of `err`?

(99% of the time I think it'll be everyone's favorite: `pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed`)

Reply:
Oh it returns the `err` and gets logged after 🤦🏻